src/en/help-azure.md

```diff
@@ -90,24 +97,21 @@ data:    Free Trial  f717c8c1-8e5e-4d38-be7f-ed1e1c879e18  true     Enabled
 info:    account list command OK
 ```
 
-The subscription ID can now be stored in a local environment variable, by
-entering:
+In this sample,
 
 ```bash
 SUB_ID=f717c8c1-8e5e-4d38-be7f-ed1e1c879e18
 ```
```

Comment:
I included more output because Azure does not label the item in the output with "subscription ID" or "SUB_ID" or anything else easily identifiable. Adding the full output, since it was only 5 lines, seemed to me to provide a road sign to more easily find what we are referring to. I'll try to further clarify what is going on here.